app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render 'shared/header', heading:'LOGIN'%>
+ <div class="w-full flex flex-col justify-center gap-2  px-5 h-96">
+  <%= form_for(resource, as: resource_name, class: "w-96", url: session_path(resource_name)) do |f| %>
+    <div class="field mt-2">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "w-full rounded-md" , placeholder: "Enter your Email"%>
```

Comment:
The same thing applies to the login page. There are just a few things you need to touch up with your login page to match the Behance design:

### Login Page:

Kindly make sure that the login pages look exactly like the snapshot below. Please refer to the [Behance website](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for reference.

Behance | Your App
------------- | -------------
![image](https://github.com/Enning94/RoR-Budget-App/assets/38088818/74cdbd65-50f4-4446-a1d4-dcedd4db13da) | ![image](https://github.com/clin2on3mun/BudgetApp/assets/38088818/b1136a2c-99ba-459b-a8a9-124cd0570fcd)


- Kindly make sure that all buttons below the form are removed. The login button should only be at the to-right corner on the navbar.
- As in the design, please make sure that the input fields are 100% of the mobile view width and the are no right and left borders. The should be no top and bottom margins for each filed.

Reply:
Hello @KDlamini 
Thank you for reviewing My PR and providing positive reviews but on a video link the link is working well when I click on the presentation link it sending me to the loom video page https://www.loom.com/share/9ff196940ec84edd9ec04131aafce33a